AppleDev.Tool/Commands/Simulators/ListSimulatorsCommand.cs

```diff
@@ -12,15 +11,25 @@ public override async Task<int> ExecuteAsync(CommandContext context, ListSimulat
     {
         var data = context.GetData();
         
-        var s = new SimCtl();
-        var sims = await s.GetSimulatorsAsync(cancellationToken: data.CancellationToken).ConfigureAwait(false);
+        var simctl = new SimCtl();
+        var sims = await simctl.GetSimulatorsAsync(availableOnly: false, cancellationToken: data.CancellationToken).ConfigureAwait(false);
```

Comment:
Potential bug: AI seemed to really not like the fact that the default of availableOnly = true meant this logic never would have done anything for unavailable.